lgsm/functions/alert_discord.sh

```diff
@@ -5,60 +5,65 @@
 # Website: https://linuxgsm.com
 # Description: Sends Discord alert.
 
+if ! command -v jq > /dev/null; then
+    fn_print_fail_nl "Sending Discord alert: jq is missing."
+    fn_script_log_fatal "Sending Discord alert: jq is missing."
+fi
+
+EscapedServername="$(echo -n "${servername}" | jq -sRr "@json")"
```

Comment:
Do we use camel case for variable names elsewhere in the project?

Reply:
Nope, just my habit. I'll push a commit tomorrow if I have time as it's late now.